tools/StaticAnalysis/Exceptions/Az.Storage/BreakingChangeIssues.csv

```diff
@@ -1,3 +1,20 @@
 "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
 "Microsoft.Azure.Commands.Management.Storage.dll","Microsoft.Azure.Commands.Management.Storage.StorageAccount.GetAzureStorageUsageCommand","Get-AzStorageUsage","0","1050","The parameter set '__AllParameterSets' for cmdlet 'Get-AzStorageUsage' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'Get-AzStorageUsage'."
 "Microsoft.WindowsAzure.Commands.Storage.dll","Microsoft.WindowsAzure.Commands.Storage.Common.Cmdlet.NewAzureStorageContext","New-AzStorageContext","0","1060","The parameter set 'AccountNameAndKey' for cmdlet 'New-AzStorageContext' is no longer the default parameter set.","Change the default parameter for cmdlet 'New-AzStorageContext' back to 'AccountNameAndKey'."
+"Microsoft.Azure.PowerShell.Cmdlets.Storage.dll","Microsoft.WindowsAzure.Commands.Storage.Table.Cmdlet.GetAzureStorageTableStoredAccessPolicyCommand","Get-AzStorageTableStoredAccessPolicy","0","1020","The cmdlet 'Get-AzStorageTableStoredAccessPolicy' no longer has output type 'Microsoft.WindowsAzure.Storage.Table.SharedAccessTablePolicy'.","Make cmdlet 'Get-AzStorageTableStoredAccessPolicy' return type 'Microsoft.WindowsAzure.Storage.Table.SharedAccessTablePolicy'."
```

Comment:
Thgese changes in namespace are fine, as long as the properties of the returned object are the same

Reply:
Yes, this properties are same.